store/sqlstore/compliance_store.go

```diff
@@ -225,12 +227,14 @@ func (s SqlComplianceStore) MessageExport(after int64, limit int) ([]*model.Mess
 			Channels.Type AS ChannelType,
 			Users.Id AS UserId,
 			Users.Email AS UserEmail,
-			Users.Username
+			Users.Username,
+			Bots.UserId IS NOT NULL AS IsBot
```

Comment:
We have a story that converts bot to user account - [MM-15441](https://mattermost.atlassian.net/browse/MM-15441). In case such user posted messages before and after conversion, does it make sense to identify UserType of "bot" or "user" at Post level, like by checking Post.Props if it has `"from_bot":"true"`?
Just asking since I'm not particularly sure about the requirements.

Reply:
I wasn't aware of this scenario. Let's bring @crspeller (who I believe worked on bots) into the conversation for a second opinion

Reply:
Bots can be converted too/from users. So you can't guarantee stability of IsBot over time. If you need this on a post level the from_bot prop was added: https://github.com/mattermost/mattermost-server/pull/10793

Reply:
So when I perform compliance export, I should take the 'from_bot' prop into account instead of the current user type, correct?

Reply:
Well I don't know what the requirements for that export are. If you need to be accurate on a per post level over time, then yes.